File: trading/js/TickerData.js
var request = require("request");
// set parameters
// multiple values possible, just comma-separate them 
// t=trading, f=funding
const symbols = "tMKRETH,fUSD,tETHUSD"; 
// set url
const url = "https://api.deversifi.com/bfx/v2/tickers?symbols=" + symbols;
// execute request and print result
request(url, function(error, response, body) {
    console.log(JSON.parse(body));
});

File: trading/python/TickerData.py
import requests
import json

# set the url
url = "https://api.deversifi.com/bfx/v2/tickers?symbols={symbols}"

"""
set parameters

symbol
    string, trading pair

You can provide a list of symbols, optional, n>0
"""
symbols = ["tMKRETH", "fUSD", "tETHUSD"]

# set the parameters
url = url.format(
    symbols=",".join(symbols))

# standard headers
headers = {
    "Accept": "application/json",
    "Content-Type": "application/json"
}

# execute the request and load into dictionary
response = json.loads(requests.request(
    "GET", url, headers=headers).text)

# print the response from dictionary
print(json.dumps(response, indent=2))
